Functions.Templates/Templates/SqlTrigger-JavaScript/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Sample SQL Trigger Binding
+ * See https://aka.ms/sqltrigger for more information about using this binding
+ *
+ * These tasks should be completed prior to running :
+ *      1. Update "tableName" in function.json - this should be the table that is monitored for changes and triggers/invokes the function.
+ *      2. Add an app setting named "SqlConnectionString" containing the connection string
+ *          to use for the SQL connection
+ *      3. Change the bundle name in host.json to "Microsoft.Azure.Functions.ExtensionBundle.Preview" and the version to "[4.*, 5.0.0)"
```

Comment:
Make sure to add a post-GA task for updating this

Reply:
Created https://github.com/Azure/azure-functions-sql-extension/issues/872 to track that, thanks.